utils/logging.py

```diff
@@ -0,0 +1,12 @@
+from logging import getLogger
+from os import getenv
+
+
+class Logger:
+    def __init__(self):
+        logger_name = getenv("BP_LOGGING")
+        self.__logger = getLogger(logger_name) if getenv("BP_LOGGING") else None
```

Comment:
This is done to support multiple logs (maybe somebody else uses Django or other logging. The env variable would be the logger name and this would be used through the whole application if needed again.